orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -39,6 +39,46 @@ func TestBaseOrchestratorMconfigBuilder_Build(t *testing.T) {
 	expectedJitteredSyncIntervalGW1 := uint32(592)
 	expectedJitteredSyncIntervalGW2 := uint32(568)
 
+	t.Run("test shared config", func(t *testing.T) {
+		nw := configurator.Network{ID: "n1", Configs: map[string]interface{}{
+			orc8r.NetworkSentryConfig: &models.NetworkSentryConfig{
+				SampleRate:   swag.Float32(0.75),
+				UploadMmeLog: true,
+				URLPython:    "https://www.example.com/v1/api",
+				URLNative:    "https://www.example.com/v1/api",
+			},
+		}}
+		gw := configurator.NetworkEntity{
+			Type: orc8r.MagmadGatewayType,
+			Key:  "gw1",
+			Config: &models.MagmadGatewayConfigs{
+				AutoupgradeEnabled:      swag.Bool(true),
+				AutoupgradePollInterval: 300,
+				CheckinInterval:         60,
+				CheckinTimeout:          10,
+				DynamicServices:         []string{},
+				FeatureFlags:            map[string]bool{},
+			},
+		}
+		graph := configurator.EntityGraph{
+			Entities: []configurator.NetworkEntity{gw},
+		}
+
+		expected := map[string]proto.Message{
+			"shared_mconfig": &mconfig_protos.SharedMConfig{
+				SentryConfig: &mconfig_protos.SharedSentryConfig{
+					SampleRate:   0.75,
+					UploadMmeLog: true,
+					UrlPython:    "https://www.example.com/v1/api",
+					UrlNative:    "https://www.example.com/v1/api",
+				},
+			},
+		}
+		actual, err := buildSharedMConfig(&nw, &graph, "gw1")
```

Comment:
I'd say let's just forego adding the `buildSharedMConfig` function, and instead use the buildBaseOrchestrator func. You'll have to add in some non-shared_mconfig values into the expected, but that's fine + even improves the test coverage a bit